dietpi/dietpi-services

```diff
@@ -217,23 +217,24 @@ Available services:
 
 			# SSH
 			aSERVICE_NAME+=('dropbear')
-			aSERVICE_NAME+=('ssh') 				# OpenSSH Server
+			aSERVICE_NAME+=('ssh') 						# OpenSSH Server
```

Comment:
When a tab is shown like 8 spaces, all is messed up now 😄. Please revert the changes here as we have spacing aligned for 1 tab = 8 spaces on all scripts.